lib/params.go

```diff
@@ -62,6 +62,12 @@ func ListParamsFromRequest(r *http.Request) ListParams {
 	return NewListParams(r.FormValue("orderBy"), page, pageSize)
 }
 
+// HasListParamsFromRequest returns true if either "page" or "pageSize" exist as form values
+// on an http.Request pointer
+func HasListParamsFromRequest(r *http.Request) bool {
+	return r.FormValue("page") != "" || r.FormValue("pageSize") != ""
+}
+
```

Comment:
Is this being used anywhere?

Reply:
ahh it was in a previous commit, removing